util/topgen.py

```diff
@@ -37,6 +39,8 @@
 from topgen.resets import Resets
 
 # Common header for generated files
+# TODO; warnhdr below is 100 chars which exceeds the 80 char limit used by
+# clang-format. Refactor this in a separate PR.
```

Comment:
Todos should be written in more of an "infinitive" tense, and include an associated bug:
```
# TODO(###)
```

"Refactor this in a separate PR" is redundant.